tests/fixtures/project_with_requirements_in/requirements.txt

```diff
@@ -4,15 +4,37 @@
 #
 #    pip-compile
 #
-click==8.1.8
+--index-url https://pypi.nuwainfo.com/nuwa/dev/+simple/
+
```

Comment:
```suggestion
```
Looks like this was added again by mistake, I can't merge the PR in this state.

Reply:
Could you maybe enable edits from maintainers? This would avoid unnecessary back and forths, as I could then apply the changes myself.